test/Libraries/RevitIntegrationTests/RenderingAsAServiceTests.cs

```diff
@@ -9,7 +9,7 @@ namespace RevitSystemTests
     [TestFixture]
     public class RenderingAsAServiceTests : SystemTest
     {
-        [Test]
+        [Test, Category("Failure")]
```

Comment:
The changes to the 4 test cases here should have been restored with Peter's fix, I mean they are not categorized to the Failure category any more.